### DISCUSSION TITLE
Vyper does not support negative indexing

### DISCUSSION THREAD

**Discussion by zed-wong** [vyperlang/vyper#3320] [Q&A]
Just trying to get the last element from an array. But I can't find a solution to do that. Any good ideas?

**Comment by zed-wong** [comment]
Since arrays are static, so the length is known. In my case, the array is filled with zero addresses and I don't know which element is the last one. 

But the problem can be simply solved by adding another parameter. XD

**Comment by theirrationalone** [comment]
```python
arr[:-1] # complete array access
arr[-1] # last element
